### DISCUSSION TITLE
ICPSHC24-10EU-IL-1 OTA problem

### DISCUSSION THREAD

**Discussion by Chartreusito** [Koenkk/zigbee2mqtt#20346] [Devices]
I've got different hardware revisions of [ICPSHC24-10EU-IL-1](https://www.zigbee2mqtt.io/devices/ICPSHC24-10EU-IL-1.html)

One of them is stuck on firmware version [1.0.002](https://ww8.ikea.com/ikeahomesmart/releasenotes/releasenotes.html)

All other are on firmware version [2.3.086](https://ww8.ikea.com/ikeahomesmart/releasenotes/releasenotes.html)

Here are the logs when i try to OTA update the "old" one :

`debug 2023-12-22 16:01:53Received MQTT message on 'zigbee2mqtt/bridge/request/device/ota_update/check' with data '{"id":"transfo_spare_3","transaction":"4xr56-1"}'
info 2023-12-22 16:01:53Checking if update available for 'transfo_spare_3'
debug 2023-12-22 16:01:53Received Zigbee message from 'transfo_spare_3', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":1,"fileVersion":65538,"imageType":16649,"manufacturerCode":4476}' from endpoint 1 with groupID 0
info 2023-12-22 16:01:53MQTT publish: topic 'zigbee2mqtt/transfo_spare_3', payload '{"brightness":162,"last_seen":"2023-12-22T16:01:53+01:00","linkquality":94,"power_on_behavior":"on","state":"ON","update":{"installed_version":65538,"latest_version":65538,"state":"idle"},"update_available":null}'
info 2023-12-22 16:01:53No update available for 'transfo_spare_3'
info 2023-12-22 16:01:53MQTT publish: topic 'zigbee2mqtt/transfo_spare_3', payload '{"brightness":162,"last_seen":"2023-12-22T16:01:53+01:00","linkquality":94,"power_on_behavior":"on","state":"ON","update":{"installed_version":65538,"latest_version":65538,"state":"idle"},"update_available":null}'
info 2023-12-22 16:01:53MQTT publish: topic 'zigbee2mqtt/bridge/response/device/ota_update/check', payload '{"data":{"id":"transfo_spare_3","updateAvailable":false},"status":"ok","transaction":"4xr56-1"}'
debug 2023-12-22 16:01:56Received Zigbee message from 'transfo_spare_3', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":1,"fileVersion":65538,"imageType":16649,"manufacturerCode":4476}' from endpoint 1 with groupID 0
info 2023-12-22 16:01:56MQTT publish: topic 'zigbee2mqtt/transfo_spare_3', payload '{"brightness":162,"last_seen":"2023-12-22T16:01:56+01:00","linkquality":94,"power_on_behavior":"on","state":"ON","update":{"installed_version":65538,"latest_version":65538,"state":"idle"},"update_available":null}'
debug 2023-12-22 16:01:56Device 'transfo_spare_3' requested OTA
debug 2023-12-22 16:01:56Responded to OTA request of 'transfo_spare_3' with 'NO_IMAGE_AVAILABLE'`

I don't have the same clusters exposed on each device.

**Comment by user34756361233** [comment]
did you find a resolution for this problem. I have a ICPSHC24-30EU-IL-1 which is also on 1.0.002. It does not want to upgrade..
